.gitignore

```diff
@@ -1,7 +1,7 @@
 __pycache__/
 .mypy_cache/
 *.py[cod]
-
+MANIFEST
```

Comment:
I'm not sure about excluding `MANIFEST`. Where would this file be coming from?

Reply:
A `MANIFEST.in` file can be added in a project to define the list of files to include in the distribution built by the sdist command.

When sdist is run, it will look for the `MANIFEST.in` file and interpret it to generate the **MANIFEST** file that contains the list of files that will be included in the package.
[some sugessions](https://perlmaven.com/dont-keep-generated-files-in-version-control) to exclude these files to be in repository